types/weixin-app/index.d.ts

```diff
@@ -0,0 +1,2160 @@
+// Type definitions for wx-app 1.0
+// Project: https://mp.weixin.qq.com/debug/wxadoc/dev/api/
+// Definitions by: taoqf <https://github.com/taoqf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/* =================== USAGE ===================
+1. Create file xxx.ts
+2. import {} from 'weixin-app';
+ =============================================== */
+
+declare namespace wx {
+	type NoneParamCallback = () => void;
+	type OneParamCallback = (data: any) => void;
```

Comment:
This is used in only one place; just inline it and provide a type to `data`. I would say that all of these type aliases should be inlined.